### ISSUE TITLE
Linter not highlighting errors with JS variables

### ISSUE THREAD

**Issue by jjmax75** [atom-community/linter#900]
I am using Linter with linter-jshint.

When I declare a variable twice no error is recorded

```var horse = 1;
var horse = 2;```

Also when a variable is undeclared no error is shown
Example
```cow = "moo";```

![test js - atom --config - atom_004](https://cloud.githubusercontent.com/assets/6083490/10066585/9de7c052-62be-11e5-8a34-b3b919f3c87b.png)

**Comment by steelbrain** [comment]
Does the output differ from the CLI output of jshint? If it does not, then open a bug over to the jshint repo :)

**Comment by jjmax75** [comment]
Thanks, I checked that and the same errors are showing but not the variable errors, I'll move to the jshint repo

**Comment by jjmax75** [comment]
Hi,

I think I found a solution to the issue.

This page (http://jshint.com/docs/options/) describes options that can be set for JSHint.

I think I specifically need to set - "undef" and "latedef" to True.

The only place in the Linter files that I can see to set these is in the .atom/packages/linter-jshint/node_modules/jshint/src/options.js file.

I've also tried setting these in config.cson but neither way has worked for me.

Is there a file I can edit to get these options enabled in JSHint?

**Comment by steelbrain** [comment]
`linter-jshint` supports `.jshintrc`. Side Note: Even if it was an issue, it would belong to the linter-jshint repo, not the base linter repo :)

**Comment by jjmax75** [comment]
Apologies, couldn't find another way to ask the question

**Comment by steelbrain** [comment]
No it's all fine, when you goto the settings of a package in atom and expand that package's settings, I believe it has a button `Report Issue` button, that might come in handy next time.

**Comment by jjmax75** [comment]
Thanks
